### DISCUSSION TITLE
Latest releases are pending

### DISCUSSION THREAD

**Discussion by kevinpapst** [tobybatch/kimai2#524] [General]
The latest Kimai releases did not make it into the Docker repository yet.
There is a problem with 32-bit ARM builds, see #525 and the fix at #526

@tobybatch is taking some time-off and will be back mid of August.

No need to ask for the "status" until then.

**Comment by dominikjas** [comment]
Thanks for the feedback! 🙂

**Comment by nemchik** [comment]
@kevinpapst linuxserver.io has also released an image https://github.com/linuxserver/docker-kimai which pulls from https://github.com/kimai/kimai/releases/latest and currently has ~~2.0.29~~ the latest releases
